content/de/docs/concepts/architecture/master-node-communication.md

```diff
@@ -41,19 +41,19 @@ Der zweite ist vom Apiserver zu einem beliebigen Node, Pod oder Dienst über die
 
 Die Verbindungen vom Apiserver zum Kubelet werden verwendet für:
 
-  * Abrufen von Protokollen für Pods.
-  * Verbinden (durch kubectl) mit laufenden Pods.
-  * Bereitstellung der Portweiterleitungsfunktion des kubelet.
+  * Das Abrufen von Protokollen für Pods.
+  * Das Verbinden (durch kubectl) mit laufenden Pods.
+  * Die Bereitstellung der Portweiterleitungsfunktion des kubelet.
 
 Diese Verbindungen enden am HTTPS-Endpunkt des kubelet.
-Standardmäßig überprüft der Apiserver das Serverzertifikat des Kubelet nicht, was die Verbindung von angreifbar für Man-in-the-Middle-Angriffe macht, und ist es ist daher **unsicher** wenn die Verbindungen über nicht vertrauenswürdige und/oder öffentliche Netzwerke laufen.
+Standardmäßig überprüft der Apiserver das Serverzertifikat des Kubelets nicht, was die Verbindung angreifbar für Man-in-the-Middle-Angriffe macht. Die Kommunikation ist daher **unsicher**, wenn die Verbindungen über nicht vertrauenswürdige und/oder öffentliche Netzwerke laufen.
 
-Um diese Verbindung zu überprüfen, verwenden Sie das Flag `--kubelet-certificate-authority`, um dem Apiserver ein Stammzertifikatbündel bereitzustellen, das zur Überprüfung des Server-Zertifikats des kubelet verwendet wird.
+Um diese Verbindung zu überprüfen, verwenden Sie das Flag `--kubelet-certificate-authority`, um dem Apiserver ein Stammzertifikatbündel bereitzustellen, das zur Überprüfung des Server-Zertifikats des kubelets verwendet wird.
 
 Wenn dies nicht möglich ist, verwenden Sie [SSH tunneling](/docs/concepts/architecture/master-node-communication/#ssh-tunnels)
-zwischen dem Apiserver und kubelet, falls erforderlich, um eine Verbindung über ein nicht vertrauenswürdiges oder öffentliches Netz zu vermeiden.
+zwischen dem Apiserver und dem kubelet, falls es erforderlich ist eine Verbindung über ein nicht vertrauenswürdiges oder öffentliches Netz zu vermeiden.
 
-Ausserdem sollte, [Kubelet Authentifizierung und/oder Autorisierung](/docs/admin/kubelet-authentication-authorization/) sollte aktiviert sein, um die kubelet-API zu sichern.
+Außerdem sollte [Kubelet Authentifizierung und/oder Autorisierung](/docs/admin/kubelet-authentication-authorization/) aktiviert sein, um die kubelet-API zu absichern.
```

Comment:
zu sichern oder abzusichern aber nicht zu absichern :)

Reply:
Right :)

Reply:
I am still thinking what I tried to do there, guess it was autocorrect (which I should turn off now :D )